pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -44,6 +45,24 @@ const (
 	webhookName = "lib_injection"
 )
 
+const (
+	webhookV1 webhookVersion = "v1"
+	webhookV2 webhookVersion = "v2"
+)
+
+type webhookVersion string
+
+func (v webhookVersion) usesInjector() bool {
+	switch v {
+	case webhookV1:
+		return false
+	case webhookV2:
+		return true
+	default:
+		return true // We're assuming future versions will use the injector as well.
+	}
```

Comment:
IMHO I think we should handle the case of unsupported version. 

WDYT?

Reply:
Yes!